src/ui/components/auth/SignUpOrResetPasswordContent.tsx

```diff
@@ -0,0 +1,129 @@
+/* Copyright (c) 2022, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import HighlightJS from "highlight.js";
+import BashHighlight from "highlight.js/lib/languages/bash";
+import { useEffect, useState } from "react";
+import { getImageUrl } from "../../../utils";
+import CopyText from "../copyText/CopyText";
+import { ContentMode } from "./types";
+
+interface ISignUpOrResetPasswordContentProps {
+	contentMode: Exclude<ContentMode, "sign-in">;
+	onBack: () => void;
+}
+const SignUpOrResetPasswordContent: React.FC<ISignUpOrResetPasswordContentProps> = ({
+	contentMode,
+	onBack,
+}: ISignUpOrResetPasswordContentProps): JSX.Element => {
+	const [showCopiedTooltip, setShowCopiedTooltip] = useState(false);
+
+	useEffect(() => {
+		HighlightJS.registerLanguage("bash", BashHighlight);
+		HighlightJS.initHighlightingOnLoad();
+	});
+
+	useEffect(() => {
+		if (showCopiedTooltip) {
+			setTimeout(() => {
+				setShowCopiedTooltip(false);
+			}, 1000);
+		}
+	}, [showCopiedTooltip]);
+
+	const getTextContent = () => {
+		switch (contentMode) {
+			case "sign-up":
+				return {
+					title: "Sign Up",
+					subtitle: "Insert the below command in your backend configuration to create a new user",
+				};
+			case "forgot-password":
+				return {
+					title: "Reset your password",
+					subtitle: "Run the below command in your terminal",
+				};
+			default:
+				throw Error("No content found for the prop!");
+		}
+	};
+
+	const { title, subtitle } = getTextContent();
+
+	const command = `curl --location --request POST 'http://localhost:3567/recipe/user/passwordhash/import' \
```

Comment:
The command isnt the same for sign up and forgot password. Also you cant hardcode `http://localhost:3567`, you need to pick this up from the core connection uri that the backend sets to the window object